StortSpelProjekt/Game/src/Gamefiles/Player.h

```diff
@@ -7,6 +7,7 @@ class UpgradeManager;
 
 struct UForUpgrade;
 struct ResetGame;
+struct SceneChange;
```

Comment:
Why did u add this?

Reply:
Early attempt of solution and unecessary, but fixed now